doc/maintainers_emeritus.rst

```diff
@@ -20,14 +22,22 @@
 - Wei Li
 - Paolo Losi
 - Gilles Louppe
+- Jan Hendrik Metzen
 - Vincent Michel
 - Jarrod Millman
+- Vlad Niculae
 - Alexandre Passos
 - Fabian Pedregosa
 - Peter Prettenhofer
+- Hanmin Qin
 - (Venkat) Raghav, Rajagopalan
 - Jacob Schreiber
+- scikit-learn-bot
```

Comment:
this one is still very active 😄

Reply:
Ah, shit, some noise created by the script. Let me fix

Reply:
Fixed